Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/dialogs/GameDetailsDialog.java

```diff
@@ -34,31 +35,31 @@ public static GameDetailsDialog newInstance(String gamePath)
     return fragment;
   }
 
+  @NonNull
   @Override
   public Dialog onCreateDialog(Bundle savedInstanceState)
   {
     GameFile gameFile = GameFileCacheManager.addOrGet(getArguments().getString(ARG_GAME_PATH));
 
-    ViewGroup contents = (ViewGroup) getActivity().getLayoutInflater()
-            .inflate(R.layout.dialog_game_details, null);
+    DialogGameDetailsBinding binding = DialogGameDetailsBinding.inflate(getLayoutInflater());
 
-    ImageView banner = contents.findViewById(R.id.banner);
+    ImageView banner = binding.banner;
 
-    TextView textTitle = contents.findViewById(R.id.text_game_title);
-    TextView textDescription = contents.findViewById(R.id.text_description);
+    TextView textTitle = binding.textGameTitle;
+    TextView textDescription = binding.textDescription;
 
-    TextView textCountry = contents.findViewById(R.id.text_country);
-    TextView textCompany = contents.findViewById(R.id.text_company);
-    TextView textGameId = contents.findViewById(R.id.text_game_id);
-    TextView textRevision = contents.findViewById(R.id.text_revision);
+    TextView textCountry = binding.textCountry;
+    TextView textCompany = binding.textCompany;
+    TextView textGameId = binding.textGameId;
+    TextView textRevision = binding.textRevision;
 
-    TextView textFileFormat = contents.findViewById(R.id.text_file_format);
-    TextView textCompression = contents.findViewById(R.id.text_compression);
-    TextView textBlockSize = contents.findViewById(R.id.text_block_size);
+    TextView textFileFormat = binding.textFileFormat;
+    TextView textCompression = binding.textCompression;
+    TextView textBlockSize = binding.textBlockSize;
 
-    TextView labelFileFormat = contents.findViewById(R.id.label_file_format);
-    TextView labelCompression = contents.findViewById(R.id.label_compression);
-    TextView labelBlockSize = contents.findViewById(R.id.label_block_size);
+    TextView labelFileFormat = binding.labelFileFormat;
+    TextView labelCompression = binding.labelCompression;
+    TextView labelBlockSize = binding.labelBlockSize;
```

Comment:
I think it would be nicer to skip defining all these variables. Most of them only get used once or twice, and not having to write `binding.` isn't much of a gain.